### ISSUE TITLE
ipad: does not work on iOS 8

### ISSUE THREAD

**Issue by phloxic** [flowplayer/flash#266]
See: https://flowplayer.org/forum/#!/open_word:ios-8-upgrade

**Comment by xnejp03** [comment]
I'm not sure if it's related to the iOS 8 update, but with the dev code it works correctly in Safari on iPad, but I don't see the Play overlay button (instead it shows the iOS control bar, but cannot press play anyway) when starting the page from the Home Screen.

To reproduce this, go to the page with the player in Safari, click on Share icon, Add to Home Screen, then go back to the Home screen and start the 'app'.

**Comment by phloxic** [comment]
Go to which page? click on which Share icon? ... You've lost me utterly and completely, sorry.

**Comment by xnejp03** [comment]
add a page with the player to the Home Screen, like this: https://www.youtube.com/watch?v=h_nGNMyiW8o

It might be related to iOS bug though: http://stackoverflow.com/questions/26985074/ios-8-homescreen-web-applications-html5-video

**Comment by phloxic** [comment]
Ah, got it, sorry for being slow on the uptake. And thanks for the link.
@xnejp03 - I have persistent trouble with my iOS devices, and all works fine in the iOS simulator. Could you just test whether this is replicable with a plain video tag, like at http://demos.flowplayer.org/videotest/minimal.html ? Then we'll know the culprit.

**Comment by xnejp03** [comment]
You will have to add these tags to the head of the page:

```
 <meta name="apple-mobile-web-app-capable" content="yes" />
 <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
```

as specified here: https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

Otherwise it just opens Safari with the page - instead of masking it as an app.

**Comment by phloxic** [comment]
@xnejp03 - there's now:
http://demos.flowplayer.org/videotest/ios-web-app.html (plain video tag)
http://demos.flowplayer.org/basics/ios-web-app.html (flowplayer)
In the simulator I get problems in the iOS simulator in both demos.

**Comment by xnejp03** [comment]
Yep, plays in Safari, but from the web app it loads the player, seems to be buffering but doesnt actually play anything.

Looks like ios bug then.
